app/models.py

```diff
@@ -404,9 +404,23 @@ def get_default_sms_sender(self):
         """
         service_sms_senders is a back reference from the ServiceSmsSender table.
         """
+        # there should only be one default sms sender per service
```

Comment:
I know you were asked to change this, but checking the default reply_to_address in the following methods is now not in the same code style as this one. Perhaps best to change it now?